builtin/stash--helper.c

```diff
@@ -603,6 +609,31 @@ static int branch_stash(int argc, const char **argv, const char *prefix)
 	return ret;
 }
 
+static int list_stash(int argc, const char **argv, const char *prefix)
+{
+	int ret;
+	struct argv_array args = ARGV_ARRAY_INIT;
+	struct option options[] = {
+		OPT_END()
+	};
+
+	argc = parse_options(argc, argv, prefix, options,
+		git_stash_helper_list_usage, PARSE_OPT_KEEP_UNKNOWN);
+
+	if (!ref_exists(ref_stash)) {
+		fprintf(stderr, "No stash entries found.");
+	}
+
+	argv_array_pushl(&args, "log", "--format=%gd: %gs", "-g", "--first-parent",
+		"-m", NULL);
+	argv_array_pushv(&args, argv);
+	argv_array_push(&args, ref_stash);
+	argv_array_push(&args, "--");
+	ret = cmd_log(args.argc, args.argv, prefix);
```

Comment:
I added it to the TODO list. I also hope that it will be done by the end of the summer, but even if it is not, there is no problem because I will continue to work on this.